src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/curatedcontent/CuratedMainPagePageObject.java

```diff
@@ -0,0 +1,116 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.curatedcontent;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.BasePageObject;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.TimeoutException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.FindBys;
+
+import java.util.List;
+
+/**
+ * @ownership: Content X-Wing
+ */
+public class CuratedMainPagePageObject extends BasePageObject {
+
+  @FindBy(css = ".mobile-top-leaderboard")
+  private WebElement mobileTopLeaderboard;
+  @FindBy(css = ".article-title")
+  private WebElement articleTitle;
+  @FindBy(css = ".wikia-stats-container")
+  private WebElement wikiaStatsContainer;
+  @FindBy(css = ".featured-content")
+  private WebElement featuredContent;
+  @FindBy(css = ".curated-content")
+  private WebElement curatedContent;
+  @FindBy(css = ".mobile-in-content")
+  private WebElement mobileInContent;
+  @FindBy(css = ".trending-articles")
+  private WebElement trendingArticles;
+  @FindBy(css = ".trending-videos")
+  private WebElement trendingVideos;
+  @FindBy(css = ".mobile-prefooter")
+  private WebElement mobilePrefooter;
+
+  @FindBys(@FindBy(css = ".curated-content-item .clamp"))
+  private List<WebElement> itemDisplayNameList;
+
+  public void verifyItem(String itemDisplayName) {
+    verifyTextInListElements(itemDisplayNameList, itemDisplayName);
+  }
+
+  public CuratedContentPageObject clickOnItem(String itemDisplayName) {
+    clickByListElementText(itemDisplayNameList, itemDisplayName);
+    return new CuratedContentPageObject(driver);
+  }
+
+  private enum Settings {
+    TIME_OUT_IN_SEC(5),
+    CHECK_OUT_IN_MILLI_SEC(1000);
+
+    private int value;
+
+    Settings(int value) {
+      this.value = value;
+    }
+  }
+
+  public CuratedMainPagePageObject(WebDriver driver) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this constructor to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)